cli/tracker.php

```diff
@@ -35,9 +33,83 @@
 /* @type Composer\Autoload\ClassLoader $loader */
 $loader->add('Application', __DIR__);
 
+// Wrap in a try/catch so we can display an error if need be
+try
+{
+	$container = (new Joomla\DI\Container)
+		->registerServiceProvider(new JTracker\Service\CliApplicationProvider)
+		->registerServiceProvider(new JTracker\Service\ConfigurationProvider)
+		->registerServiceProvider(new JTracker\Service\DatabaseProvider)
+		->registerServiceProvider(new JTracker\Service\DebuggerProvider)
+		->registerServiceProvider(new JTracker\Service\DispatcherProvider)
+		->registerServiceProvider(new JTracker\Service\GitHubProvider)
+		->registerServiceProvider(new JTracker\Service\MonologProvider)
+		->registerServiceProvider(new JTracker\Service\TransifexProvider)
+		->registerServiceProvider(new JTracker\Service\WebApplicationProvider)
+		->registerServiceProvider(new Application\Service\LoggerProvider);
+
+	// Create the application aliases for the common 'app' key and base application class
+	$container->alias('Joomla\\Application\\AbstractApplication', 'Application\\Application')
+		->alias('app', 'Application\\Application');
+
+	// Create the logger aliases for the common 'monolog' key, the Monolog Logger class, and the PSR-3 interface
+	$container->alias('monolog', 'monolog.logger.cli')
+		->alias('Monolog\\Logger', 'monolog.logger.cli')
+		->alias('Psr\\Log\\LoggerInterface', 'monolog.logger.cli');
+}
+catch (\Exception $e)
+{
+	if (isset($container))
+	{
+		// Try to write to a log
+		try
+		{
+			$logger = $container->get('monolog.logger.cli');
+			$logger->critical(
+				sprintf(
+					'Exception of type %1$s thrown while booting the application',
+					get_class($e)
+				),
+				['exception' => $e]
+			);
+		}
+		catch (\Exception $nestedException)
+		{
+			// Do nothing, we tried our best
+		}
```

Comment:
IMO it's not a big concern here as the nested exception would be an error writing to the log file.  The actual error causing the script to fail is spit out below.